src/guides/v2.4/install-gde/prereq/mysql.md

```diff
@@ -33,327 +33,94 @@ If your web server and database server are on different hosts, perform the tasks
-
-1. *CentOS 6* Install the MySQL database:
-
-   ```bash
-   yum -y update
-   ```
-
-   ```bash
-   sudo wget http://repo.mysql.com/mysql-community-release-el6-5.noarch.rpm && sudo rpm -ivh mysql-community-release-el6-5.noarch.rpm
-   ```
-
-   ```bash
-   sudo yum -y install mysql-server
-   ```
-
-1. *CentOS 7* Install the MySQL database:
-
-   ```bash
-   yum -y update
-   ```
-
-   ```bash
-   sudo wget http://repo.mysql.com/mysql-community-release-el7-5.noarch.rpm && sudo rpm -ivh mysql-community-release-el7-5.noarch.rpm
-   ```
-
-   ```bash
-   sudo yum -y install mysql-server
-   ```
-
-1. Start MySQL:
-
-   ```bash
-   service mysqld start
-   ```
-
-1. Set a password for the <tt>root</tt> user and set other security-related options. Enter the following command and follow the prompts on your screen to complete the configuration:
-
-   ```bash
-   mysql_secure_installation
-   ```
-
-1. Verify the MySQL server version:
-
-   ```bash
-   mysql -u root -p
-   ```
-
-   Messages similar to the following display:
-
-   ```terminal
-   Welcome to the MySQL monitor.  Commands end with ; or \g.
-   Your MySQL connection id is 15 Server version: 5.6.23 MySQL Community Server (GPL)
-
-   Copyright (c) 2000, 2015, Oracle and/or its affiliates. All rights reserved.
-
-   Oracle is a registered trademark of Oracle Corporation and/or its affiliates. Other names may be trademarks of their respective owners.
-
-   Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
-   ```
+Magento disabled the regular validation behavior by setting SET SQL_MODE='' in `/lib/internal/Magento/Framework/DB/Adapter/Pdo/Mysql.php:424.`. With validation disabled, it is possible that MySQL will truncate data. In MySQL, the Query behavior has changed: `Select * on my_table where IP='127.0.0.1'` will no longer return any results because the IP address is now properly seen as a string, rather than an integer.
 
-1. If you expect to import large numbers of products into Magento, you can configure MySQL to use the [`max_allowed_packet`](http://dev.mysql.com/doc/refman/5.6/en/program-variables.html){:target="_blank"} parameter. We recommend a value of at least 16MB.
+## Upgrading from MySQL 5.7 to MySQL 8
 
-   {% include install/mysql_max-allowed-packet-centos.md %}
+To properly update MySQL from version 5.7 to version 8, you must follow these steps in order:
 
-1. Configure the Magento database instance as discussed in the next section.
+1. Upgrade Magento to 2.4.0.
+   Test everything and make sure your system works as expected.
+1. Enable maintenance mode: `bin/magento maintenance:enable`.
+1. Make a data [backup]({{page.baseurl }}/install-gde/install/cli/install-cli-backup).html#instgde-cli-uninst-back) from the database.
+1. Update MySQL to version 8.
+1. Import the backed-up data into MySQL.
+1. Set the database engine in the Magento admin to MySQL 8.
```

Comment:
Is there a `bin/magento` command for that?

Reply:
I can't find one.

Reply:
See https://devdocs.magento.com/guides/v2.3/reference/cli/magento.html#setupinstall